specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/privateLinkService.json

```diff
@@ -740,9 +755,9 @@
           "$ref": "./virtualNetwork.json#/definitions/Subnet",
           "description": "The reference of the subnet resource."
         },
-        "publicIPAddress": {
```

Comment:
This should a breaking change in SDK.
In PowerShell I think you could just mark publicIpAddress parameter as deprecated and remove it later

Reply:
Yes, will do that in powershell. Thank you.